src/state/policy.rs

```diff
@@ -57,12 +79,14 @@ impl PolicyStore {
             RbacScope::WorkloadSelector => {}
         }
         self.by_key.insert(key, rbac);
+        self.notifier.send();
```

Comment:
Probably we want to send at most 1 per xds push. That may modify many policies at once

Reply:
I was going to look to a follow up which adds batching and includes putting the scope onto the channel so that the receivers have less spam and can inspect the scope to determine if they need to re-assert. If we think it's critical path I could include it but if not critical perhaps a TODO would be better so it's not just in my head.

Reply:
added an impl for your comment. Not scoped but should reduce the number of notifications sent to policy subscribers.